src/modules/economy/components/AccountCard.tsx

```diff
@@ -78,23 +62,75 @@ export const AccountCard: React.VFC<AccountCardProps> = ({ account }) => {
   }
 
   return (
-    <Card>
-      <Card.Section m="xs">
-        <Label>Saldo:</Label>
-        <Balance>{numberWithSpaces(account.balance)},- NOK</Balance>
-      </Card.Section>
-      <Card.Section m="xs">
-        <Label>Kortnummer:</Label>
-        <CardInputContainer>
-          <input
-            value={cardUuid}
-            disabled={!editable}
-            onChange={evt => setCardUuid(evt.target.value)}
-            onBlur={toggleEditable}
-          />
-          <EditIcon onClick={toggleEditable} />
-        </CardInputContainer>
-      </Card.Section>
+    <Card withBorder className={classes.balanceCard}>
+      <Group position={'right'}>
+        <Title
+          order={4}
+          variant={'gradient'}
+          gradient={{ from: 'samfundet-red.0', to: 'samfundet-red.2', deg: 30 }}
+        >
+          Socibanken
+        </Title>
+      </Group>
+      <Card className={classes.cardInner}>
+        <Group position={'apart'}>
+          <Card.Section p={'lg'}>
+            <Badge color={'samfundet-red'} size={'sm'}>
+              Saldo
+            </Badge>
+            <Title align="center" weight={'lighter'} color={'gray.0'}>
+              {numberWithSpaces(account.balance)} kr
+            </Title>
+          </Card.Section>
+          <Button
+            component={Link}
+            to={'/economy/deposits/create'}
+            variant={'light'}
+            color={'samfundet-red'}
+          >
+            <IconCash />
+            <IconPlus />
+          </Button>
+        </Group>
+        <Group position={'center'}>
+          <Card.Section>
+            <Text align="center">Kortnummer</Text>
+            <Group position="center">
+              <TextInput
+                value={cardUuid}
+                disabled={!editable}
+                onChange={evt => setCardUuid(evt.target.value)}
+                onBlur={toggleEditable}
+              />
+              <ActionIcon onClick={toggleEditable}>
+                <IconEdit stroke={1.4} color="white" />
+              </ActionIcon>
+            </Group>
+          </Card.Section>
+        </Group>
+      </Card>
     </Card>
   )
 }
+
+const useStyles = createStyles(theme => ({
+  cardWithBorder: {
+    borderTop: `5px solid ${theme.colors.brand}`,
+  },
+  balanceCard: {
+    backgroundImage: theme.fn.gradient({
+      from: 'samfundet-red.8',
+      to: 'samfundet-red.4',
+      deg: 30,
+    }),
+    color: theme.white,
+    maxWidth: 450,
+    maxHeight: 300,
+    borderRadius: theme.radius.lg,
+  },
+  cardInner: {
+    backgroundColor: 'rgba(255,255,255,0.03)',
+    borderRadius: theme.radius.lg,
+    color: theme.white,
+  },
```

Comment:
Get rid of the 'extra' border within the card

Reply:
Fiex